<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>17</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>1/3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>50</y>
          <width>174</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Reservation Name: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>197</y>
          <width>97</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Room Type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>223</y>
          <width>116</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Room Choice:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>271</y>
          <width>164</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Reservation Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>319</y>
          <width>232</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Length of Stay (nights):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>367</y>
          <width>164</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Number of Adults:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>415</y>
          <width>184</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Number of Children:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>487</y>
          <width>232</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pre-tax Estimated Total:</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="nameInput">
        <property name="geometry">
         <rect>
          <x>262</x>
          <y>40</y>
          <width>511</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Sans Typewriter</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QComboBox" name="roomSize">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>197</y>
          <width>62</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="roomView">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>245</y>
          <width>62</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="resDate">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>293</y>
          <width>78</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="lenStay">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>341</y>
          <width>33</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="numAdults">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>389</y>
          <width>33</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="numChild">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>437</y>
          <width>33</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="parking">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>463</y>
          <width>59</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Parking</string>
        </property>
       </widget>
       <widget class="QLabel" name="totCost">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>509</y>
          <width>114</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>0</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>2/3</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page3">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>0</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>3/3</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
